db/infinispan/src/main/java/org/commonjava/indy/infinispan/data/InfinispanStoreDataReverseMapStartupAction.java

```diff
@@ -0,0 +1,44 @@
+package org.commonjava.indy.infinispan.data;
+
+import org.commonjava.indy.action.IndyLifecycleException;
+import org.commonjava.indy.action.StartupAction;
+import org.commonjava.indy.data.IndyDataException;
+import org.commonjava.indy.data.StoreDataManager;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class InfinispanStoreDataReverseMapStartupAction
```

Comment:
One concern for this startup action: will this be conflict with InfinispanStoreDataManager.init()? I think the init() also did some initialization on the reverse-map.

Reply:
So, this is a great point. It probably indicates that we found a different bug with the new functional test, when we used the migration action as a data-loading mechanism. I've moved both by-pkg and affected-by out to startup actions, so they'll work with migrated data as well as stuff we already have stored.

I'm going to work on a different test setup that doesn't use the migration action to load the initial repo definitions, and see if I can get it to break.